### PR TITLE
TT-664 fix bad UX

### PR DESCRIPTION
**Pull request by ngalvare10**

### CODE REVIEW

File: src/app/modules/reports/components/time-entries-table/time-entries-table.component.scss [hunk 1]
```diff
@@ -132,3 +132,5 @@ table.dataTable thead .sorting_desc {
   font-style: italic;
   cursor: pointer;
 }
+
+
```
Comment (nicolsss) [inline src/app/modules/reports/components/time-entries-table/time-entries-table.component.scss:136]: Just leave one space after the last line

File: src/app/modules/reports/components/time-entries-table/time-entries-table.component.ts [hunk 2]
```diff
@@ -88,28 +92,33 @@ export class TimeEntriesTableComponent implements OnInit, OnDestroy, AfterViewIn
   ngOnInit(): void {
     this.rerenderTableSubscription = this.reportDataSource$.subscribe((ds) => {
       this.sumDates(ds.data);
-      this.rerenderDataTable();
     });
+
```
Comment (nicolsss) [inline src/app/modules/reports/components/time-entries-table/time-entries-table.component.ts:96]: Remove whitespace

File: src/app/modules/reports/components/time-entries-table/time-entries-table.component.ts [hunk 3]
```diff
@@ -88,28 +92,33 @@ export class TimeEntriesTableComponent implements OnInit, OnDestroy, AfterViewIn
   ngOnInit(): void {
     this.rerenderTableSubscription = this.reportDataSource$.subscribe((ds) => {
       this.sumDates(ds.data);
-      this.rerenderDataTable();
     });
+    
     this.uploadUsers();
+
```
Comment (nicolsss) [inline src/app/modules/reports/components/time-entries-table/time-entries-table.component.ts:98]: remove whitespace

File: src/app/modules/reports/components/time-entries-table/time-entries-table.component.ts [hunk 4]
```diff
@@ -88,28 +92,33 @@ export class TimeEntriesTableComponent implements OnInit, OnDestroy, AfterViewIn
...
-      this.rerenderDataTable();
     });
+    
     this.uploadUsers();
+    
   }
 
   ngAfterViewInit(): void {
     this.rerenderDataTable();
+
```
Comment (nicolsss) [inline src/app/modules/reports/components/time-entries-table/time-entries-table.component.ts:103]: remove whitespace

File: src/app/modules/reports/components/time-entries-table/time-entries-table.component.ts [hunk 5]
```diff
@@ -88,28 +92,33 @@ export class TimeEntriesTableComponent implements OnInit, OnDestroy, AfterViewIn
...
+  private rerenderDataTable(): any {
     if (this.dtElement && this.dtElement.dtInstance) {
       this.dtElement.dtInstance.then((dtInstance: DataTables.Api) => {
         dtInstance.destroy();
         this.dtTrigger.next();
       });
     } else {
+      
       this.dtTrigger.next();
+
```
Comment (nicolsss) [inline src/app/modules/reports/components/time-entries-table/time-entries-table.component.ts:120]: remove whitespace

File: src/app/modules/reports/components/time-entries-table/time-entries-table.component.ts [hunk 6]
```diff
@@ -151,6 +160,4 @@ export class TimeEntriesTableComponent implements OnInit, OnDestroy, AfterViewIn
   user(userId: string){
     this.selectedUserId.emit(userId);
   }
-
-}
-
+}
```
Comment (nicolsss) [inline src/app/modules/reports/components/time-entries-table/time-entries-table.component.ts:163]: Put one space after the last line

**Review by nicolsss — APPROVED** [review]

File: src/app/modules/reports/components/time-entries-table/time-entries-table.component.ts [hunk 7]
```diff
@@ -88,28 +92,28 @@ export class TimeEntriesTableComponent implements OnInit, OnDestroy, AfterViewIn
   ngOnInit(): void {
     this.rerenderTableSubscription = this.reportDataSource$.subscribe((ds) => {
       this.sumDates(ds.data);
-      this.rerenderDataTable();
     });
-    this.uploadUsers();
+    this.uploadUsers();
```
Comment (sbateca) [inline src/app/modules/reports/components/time-entries-table/time-entries-table.component.ts:96]: Delete the whitespace

File: src/app/modules/reports/components/time-entries-table/time-entries-table.component.ts [hunk 8]
```diff
@@ -88,28 +92,28 @@ export class TimeEntriesTableComponent implements OnInit, OnDestroy, AfterViewIn
...
       this.sumDates(ds.data);
-      this.rerenderDataTable();
     });
-    this.uploadUsers();
+    this.uploadUsers();  
   }
 
   ngAfterViewInit(): void {
-    this.rerenderDataTable();
+    this.rerenderDataTable();
```
Comment (sbateca) [inline src/app/modules/reports/components/time-entries-table/time-entries-table.component.ts:100]: Delete the whitespace

File: src/app/modules/time-entries/pages/time-entries.component.html [hunk 9]
```diff
@@ -1,18 +1,16 @@
 <div>
     <div>
         <button type="button" (click)="newEntry()" data-toggle="modal" data-target="#editRecordsByDate" class="btn btn-primary">
-      Add new entry
-    </button>
-        <button type="button" (click)="onDisplayModeChange()" class="btn btn-primary float-right" *ngIf="isFeatureToggleCalendarActive">
-      <em class="fas fa-list" *ngIf="displayGridView"></em>
-      <em class="fas fa-th" *ngIf="!displayGridView"></em>
-    </button>
+        Add new entry
```
Comment (sbateca) [inline src/app/modules/time-entries/pages/time-entries.component.html:4]: Review the whitespaces and the indent

**Pull request closed without merging**